plumbum/cli/switches.py

```diff
@@ -464,18 +464,47 @@ class MyApp(Application):
                              comparison or not. The default is ``False``
     :param csv: splits the input as a comma-separated-value before validating and returning
                 a list. Accepts ``True``, ``False``, or a string for the separator
-    """
+    :param all_markers: when a user inputs any value from this set, it is considered that
+                        he requested iteration over all values. By default `all_markers`
```

Comment:
```suggestion
                        they requested iteration over all values. By default `all_markers`
```

Reply:
https://stallman.org/articles/genderless-pronouns.html

Reply:
```suggestion
                        to be a request for iteration over all values. By default `all_markers`
```

Reply:
(and also remove "that" from the line above)